website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -25,8 +25,11 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
 @include 'tfc-package-callouts/ephemeral-workspaces.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
```

Comment:
Do users need specific permissions to perform auto destroy runs? I realize the original page didn't have that info, but I think it'd be good to have here!

Reply:
Also I'm sorry to be annoying but could you do a find and replace and replace "auto destroy" with "auto-destroy" everywhere on this page? I apologize for making you do this, but I definitely think it's worth it to fix! 🙏

Reply:
> Do users need specific permissions to perform auto destroy runs? I realize the original page didn't have that info, but I think it'd be good to have here!

Added this, let me know if it should be changed at all
```md
!> **Note:** Automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend only using this setting for development environments.
```

> Also I'm sorry to be annoying but could you do a find and replace and replace "auto destroy" with "auto-destroy" everywhere on this page? I apologize for making you do this, but I definitely think it's worth it to fix! 🙏

Done! Any issues with either how it's display on the page right now (like button text-wise) or with there being a disconnect between the two?

<img width="887" alt="image" src="https://github.com/hashicorp/terraform-docs-common/assets/33099255/c10cb91c-9b81-4ee0-a763-854c5526b495">

Reply:
...well now I want to put in a request to update that UI copy. I think auto-destroy reads more clearly as a compound word. 

This is definitely my fault for not catching this the first time 😅 Do you happen to know who I should talk to or where I should request that change? No worries if not! I can try to figure it out

Reply:
@Ashtronaut09 any issues with changing all of the UI copy from `auto destroy`->`auto-destroy`? It should be a pretty easy change if everything's all good.

Reply:
Updating UI copy as part of #17767